readthedocs/invitations/models.py

```diff
@@ -200,8 +217,49 @@ def object_name(self):
     def object_url(self):
         return self.backend.get_object_url()
 
+    @property
+    def audit_data(self):
```

Comment:
We could also use serializers for each model, but we only do this for 3 models so far, maybe this is fine for now.

Reply:
I went ahead and used serializers, they are easier to implement.